doc/release-notes/rl-2105.adoc

```diff
@@ -50,8 +50,8 @@ As a result of this change, <<opt-programs.mpv.package>> is no longer the
 resulting derivation. Use the newly introduced `programs.mpv.finalPackage`
 instead.
 
-* The <<opt-programs.rofi.extraConfig>> option is now an attrset rather
-than a string. To migrate, move the each line into the attrset,
+* The <<opt-programs.rofi.extraConfig>> option is now an attribute set rather
+than a string. To migrate, move the each line into the attribute set,
```

Comment:
```suggestion
than a string. To migrate, move each line into the attribute set,
```

The `rofi` prefix needs to be dropped too, right?

Reply:
Yeah, it is right below.